### DISCUSSION TITLE
STATUS_CODES_WITH_NO_BODY should also include 205.

### DISCUSSION THREAD

**Discussion by declaresub** [fastapi/fastapi#8642] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi.openapi.constants import STATUS_CODES_WITH_NO_BODY
assert 205 in STATUS_CODES_WITH_NO_BODY
```

### Description

An HTTP response with status code 205 Reset Content must not contain a message body; see [RFC7231 Section 6.3.6](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.6).  So fastapi.openapi.constants.STATUS_CODES_WITH_NO_BODY should contain 205.

### Operating System

Linux, macOS

### FastAPI Version

0.63.0

### Python Version

3.9.5

**Comment by YuriiMotov** [comment]
It's done in current version
